stories/range-slider.stories.js

```diff
@@ -28,12 +20,8 @@ storiesOf('Refinements|RangeSlider', module)
         instantsearch.widgets.rangeSlider({
           container,
           attribute: 'price',
-          step: 500,
-          tooltips: {
-            format(rawValue) {
-              return `$${Math.round(rawValue).toLocaleString()}`;
-            },
-          },
+          step: 100,
+          pips: false,
```

Comment:
I guess since pips is the number, but we don't know the range is why they can't line up with step?

Reply:
Yes, I explained in the post.